frontend/src/lib/i18n/locales/en.json

```diff
@@ -156,6 +156,7 @@ the [Linguistics Institute at Payap University](https://li.payap.ac.th/) in Chia
     "table": {
       "name": "Name",
       "code": "Code",
+      "created_at": "Created",
       "last_change": "Last Change",
```

Comment:
If you feel like it, you can fix this label. I believe it's actually the Last Commit timestamp. (or the deleted date if it's deleted, but in a way, the deleting is the last commit, so I think "Last Commit" is fitting).

Reply:
I feel like "Last Change" is fine here, as we don't actually need to convey internal details about the implementation. E.g., if someday we're using CRDTs instead of Mercurial, would we still want to call it "Last Commit"? Whereas "Last Change" still conveys the right meaning ("someone changed the project" i.e. modified its data) regardless of the backend data-storage we're using. So I'm inclined to leave this one as-is.